### PR TITLE
PodSecurityPolicy example: fix explanation

### PR DESCRIPTION
**Pull request by ericchiang**
cc @kubernetes/sig-auth-pr-reviews

```release-note
NONE
```

### CODE REVIEW

File: examples/podsecuritypolicy/rbac/README.md [hunk 1]
```diff
@@ -82,10 +82,12 @@ podsecuritypolicy "restricted" created
 
 ### Roles and bindings
 
-In order to a `PodSecurityPolicy` a user must have the ability to perform the `use` verb on the policy.
-The `use` verb is a special verb that grants access to use the policy while
-not allowing any other access.  This verb is specific to `PodSecurityPolicy`.
-To enable the `use` access we will create cluster roles.  In this example we will provide the roles:
+In order to create a pod with a `securityContext`, a user must have the ability
```
Comment (liggitt) [inline examples/podsecuritypolicy/rbac/README.md]: It actually applies to all pods, and to fields outside the `securityContext`. I'd suggest

> In order to create a pod, either the creating user or the service account specified by the pod must be authorized to use a `PodSecurityPolicy` object that allows the pod. That authorization is determined by the ability to perform the `use` verb on a particular `podsecuritypolicies` resource. The `use` verb is a special verb that grants access to use the policy while not allowing any other access. To enable the `use` access we will create cluster roles.  In this example we will provide the roles:

**Comment by ericchiang** [comment]
@liggitt updated.

**Comment by liggitt** [comment]
this should move into the doc repo at some point... approving here for now

**Pull request merged**